test/visibility/import/edgeCases/multipleDef3.good

```diff
@@ -1,2 +1,4 @@
 multipleDef3.chpl:2: error: symbol x is multiply defined
 multipleDef3.chpl:5: note: also defined here
+multipleDef3.chpl:2: error: symbol x is multiply defined
+multipleDef3.chpl:5: note: also defined here
```

Comment:
This multiple error case is a bit different, and I'll spend some time on trying to remove this.